index.html

```diff
@@ -3202,6 +3206,88 @@ <h3><code>securityDefinitions</code> and <code>security</code></h3>
 Using a specific prefix as in the above example
 for URI variables declared in security schemes can make it easier to avoid name conflicts.
 </p>
+<p>Security parameters might also be included along with the payload in some
+systems.  For example, suppose a system requires every payload to be a JSON
+object including a member named <code>key</code> whose value is a secret key.
+Depending on the interaction, however, other elements of the JSON object might 
+vary.  This situation can be dealt with using the <code>body</code> security
+information location.
+Note that for this location, the <code>name</code> parameter is actually a JSON pointer
+evaluated relative to the root of the <code>DataSchema</code> for each interaction
+it is bound with, which allows it to be used with payloads that vary in other respects.
+As an example, here is a light that has a property to set its brightness and color
+and two separate actions to turn it on and off.  Although the JSON payloads
+are different for these actions the <code>key</code> element occurs in the same 
+relative location so single JSON pointer can be used.
+Note: if the security key occurs in inconsistent locations, it will be
+necessary to use multiple security scheme definitions.
+<pre class="example">
+{
+    ...
+    "securityDefinitions": {
+        "apikey_body": {
+            "scheme": "apikey",
+            "in": "body",
+            "name": "/key"
```

Comment:
Should have a leading #, e.g. and maybe should also be "deeper", e.g. nested structure, like `#/auth/key`, which means changing the data schema too.

Reply:
Actually... it shouldn't, not a fragment identifier.